src/Integration/LocalServices/ErrorListInfoBarController.cs

```diff
@@ -170,13 +175,22 @@ private void CancelQualityProfileProcessing()
             this.CurrentBackgroundProcessor = null;
         }
 
-        internal /*for testing purposes*/ void ProcessSolutionBinding()
+        private void ProcessSolutionBinding()
+        {
+            logger.LogDebug("[ErrorListInfoBarController] Processing solution binding...");
+            ProcessSolutionBindingAsync().Forget();
+        }
+
+        internal/* for testing */ async Task ProcessSolutionBindingAsync()
         {
+            logger.LogDebug("[ErrorListInfoBarController] Processing solution binding...");
```

Comment:
I hadn`t noticed I was logging the same debug message twice (here and in line 180).
I'll remove one.